definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -271,56 +289,68 @@
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "CoRespondentSolicitorName",
-    "PageFieldDisplayOrder": 15,
+    "CaseFieldID": "CoRespStatementOfTruth",
+    "PageFieldDisplayOrder": 20,
     "DisplayContext": "MANDATORY",
     "PageID": "aosdataentrycoresp1",
     "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Yes\"",
+    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"No\"",
     "ShowSummaryChangeOption": "Yes"
   },
   {
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "CoRespondentSolicitorEmail",
-    "PageFieldDisplayOrder": 16,
+    "CaseFieldID": "D8ReasonForDivorceAdultery3rdAddress",
+    "PageFieldDisplayOrder": 21,
     "DisplayContext": "OPTIONAL",
     "PageID": "aosdataentrycoresp1",
     "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Yes\"",
+    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"No\"",
     "ShowSummaryChangeOption": "Yes"
   },
   {
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "CoRespondentSolicitorPhone",
-    "PageFieldDisplayOrder": 17,
+    "CaseFieldID": "CoRespEmailAddress",
+    "PageFieldDisplayOrder": 22,
     "DisplayContext": "OPTIONAL",
     "PageID": "aosdataentrycoresp1",
     "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Yes\"",
+    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"No\"",
     "ShowSummaryChangeOption": "Yes"
   },
   {
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "CoRespSolAOSSOT",
-    "PageFieldDisplayOrder": 18,
-    "DisplayContext": "MANDATORY",
+    "CaseFieldID": "CoRespPhoneNumber",
+    "PageFieldDisplayOrder": 23,
+    "DisplayContext": "OPTIONAL",
     "PageID": "aosdataentrycoresp1",
     "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Yes\"",
+    "FieldShowCondition": "CoRespondentSolicitorRepresented=\"No\"",
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "updateContactDetails",
```

Comment:
Here, just need to update the eventid to: EnterAOSOffline_CoRespondent
and pageID to: aosdataentrycoresp1

Reply:
ouch! good spot. Thanks